deploy/kubedirector/rbac-default.yaml

```diff
@@ -39,6 +39,7 @@ rules:
   - apps
   resources:
   - deployments
+  - replicasets
```

Comment:
Was this something that came from the generated SDK framework somehow, or did you find that you needed to add this?